lib/strategies/android-chrome.js

```diff
@@ -56,16 +55,20 @@ AndroidChromeStrategy.prototype._afterReformatInput = function (formattedState)
   // of the input has changed when adding
   // permacharacters. This results in the selection
   // putting the cursor before the permacharacter,
-  // instead of after. So we setTimeout and check
-  // the position of the cursor. If it does not match
-  // the formatted input, we set it again
+  // instead of after.
+  //
+  // There is also the case of some Android Chrome
+  // keyboards reporting a ranged selection on the
+  // first input. Restricted Input maintains that
```

Comment:
What does "first input" mean? Could we make this a little clearer (or am I missing something obvious)?